src/daft-core/src/array/ops/list.rs

```diff
@@ -35,6 +36,31 @@ fn join_arrow_list_of_utf8s(
         })
 }
 
+fn arrow_list_contains(
+    list_elements: impl Iterator<Item = Option<Box<dyn arrow2::array::Array>>>,
+    elements: impl Iterator<Item = Box<dyn arrow2::array::Array>>,
```

Comment:
Seems like I'm using `Box<dyn arrow2::array::Array>` as a bootleg "AnyValue" here.

Technically, a single-element Series could be used as a bootleg AnyValue too 😛

Reply:
Yeah lol